src/styles/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+
+  &__star {
+    width: 16px;
+    height: 16px;
```

Comment:
Consider defining SCSS variables for the width and height (`16px`) to ensure consistency and ease of updates.